theano/scalar/basic.py

```diff
@@ -604,12 +604,11 @@ def __sub__(self, other):
     def __mul__(self, other):
         return mul(self, other)
 
-    if PY3:
-        def __truediv__(self, other):
-            return div_proxy(self, other)
-    else:
-        def __div__(self, other):
-            return div_proxy(self, other)
```

Comment:
@nouiz, I've to test ```//``` and ```/``` on a tensor with and without future in python2 and test if the one with future behaves as python3 ?